src/Components/Payment/UserSettingsPayments.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, Serif, Theme } from "@artsy/palette"
+import { UserSettingsPayments_me } from "__generated__/UserSettingsPayments_me.graphql"
+import { UserSettingsPaymentsQuery } from "__generated__/UserSettingsPaymentsQuery.graphql"
+import { ContextConsumer, ContextProps } from "Artsy"
+import { get } from "Utils/get"
+
+import React from "react"
+import {
+  createFragmentContainer,
+  graphql,
+  QueryRenderer,
+  RelayProp,
+} from "react-relay"
+import { PaymentFormWrapper } from "./PaymentFormWrapper"
+import { SavedCreditCards } from "./SavedCreditCards"
+
+interface UserSettingsPaymentsProps extends ContextProps {
+  relay?: RelayProp
+  me: UserSettingsPayments_me
+}
+
+export class UserSettingsPayments extends React.Component<
+  UserSettingsPaymentsProps
+> {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const creditCardEdges = get(this.props, props => props.me.creditCards.edges)
+    const creditCards = creditCardEdges.map(({ node: creditCard }) => {
+      return creditCard
+    })
+
+    return (
+      <>
+        <Theme>
```

Comment:
Can likely remove Theme and outer `<>`

Reply:
This is the top-level component, so I think the `Theme` is needed  here?